pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -3430,26 +3622,6 @@ func TestGetFinallyTaskRunTimeout(t *testing.T) {
 			},
 		},
 		expected: &metav1.Duration{Duration: 15 * time.Minute},
-	}, {
-		name: "finally taskrun with elapsed time; timeouts.pipeline applies",
```

Comment:
I removed this because what it was testing for - finally tasks getting the remaining time in `timeouts.Pipeline` as their timeout, rather than `timeouts.Finally` or `PipelineTask.Timeout` - is no longer relevant, since that's exactly the behavior we're replacing with the signal approach.